docs/index.asciidoc

```diff
@@ -63,16 +64,28 @@ output plugins.
 Can Retry.
 Setting this value true helps user to send multiple retry attempts if the first request fails
 
+[id="plugins-{type}s-{plugin}-flush_size"]
+===== `flush_size`
+
+  * Value type is <<number,number>>
+  * Default value is 100
```

Comment:
since the default batch size in logstash is 125, I'd suggest either setting this default to that exact value or to fetch the configured `pipeline.batch.size` value. Otherwise we will always be splitting batches of 100/25 by default

Reply:
Good to know, I'll make this change.